src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -161,5 +180,136 @@ public Task ExecuteChildContentAsync()
 
             return _childContent;
         }
+
+        private class ItemsCopyOnWriteDictionary : IDictionary<string, object>
```

Comment:
Ignore this, it's here until we find a suitable home for common web framework utilities.